test/controllers/v1/value_definitions_controller_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module V1
+  class ValueDefinitionsControllerTest < ActionDispatch::IntegrationTest
+    setup do
+      User.current = FactoryBot.create(:user)
+      ValueDefinitionsController.any_instance.stubs(:authenticate_user).yields
+    end
+
+    should 'lists all value_definitions' do
```

Comment:
Hmm, maybe at least create a few definitions and verify if they are displayed? Just to test what you're actually writing here.